docs/program/apis/disconnect.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Session Connectivity: Network Disconnection during a Session"
+linkTitle: "Network Disconnection"
+weight: 20
+type: "docs"
+description: "How Viam handles losing connection to the internet during a robot session."
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session"]
+---
+
+What happens when your robot loses its connection to the internet and can no longer talk to the Viam cloud through [the Viam app](https://app.viam.com).
+
+### How does `viam-server` work if you lose connectivity during a session?
+
+If you lose connectivity, `viam-server` will by default timeout all resource and end this robot's [*session*](/program/apis/sessions/).
+This does not immediately stop the flow of power to all actuating resources, but it signals that the robot can no longer receive operations over the internet, so any further operations will not be performed.
+
+This is because the robot cannot access Viam's APIs, including the robot API, without a connection to the cloud.
+
+To disable the default behavior here and manage resource timeout and reconfiguration over a networking session yourself, follow [these instructions](/program/apis/sessions/) to [disable the default behavior](/program/apis/sessions/#disable-default-session-management) of session management, then utilize [Viam's SDKs](/program/) in your code to make calls to [the session management API](https://pkg.go.dev/go.viam.com/rdk/session#hdr-API).
+If you do this, and configure the primary and sub-parts of your robot to cache all tokens so that authentication to Viam's cloud is not required for operation-related communication over the local network or directly through `viam-server`, you can operate your robot consistently without the need for a network connection.
+However, the low-network approach is not recommended if you want to utilize any of Viam's [services](/services/), like the vision service, motion planning, or the frame system.
+
+### How do Viam's SDKs work if you lose connectivity during a session?
```

Comment:
```suggestion
### How do Viam's client SDKs work if you lose connectivity during a session?
```

Reply:
Again, what is "you" here? Is it the client, or the robot?

Reply:
It's a user reading-- we've been using you a lot in our docs to just talk to the reader. I can see how that introduces confusion here, though. Changing to "your robot" as I believe that is the intended use case